src/transformers/pipelines.py

```diff
@@ -815,6 +833,42 @@ def __call__(self, *args, **kwargs):
             ]
 
 
+class NLIForZeroShotPipeline(Pipeline):
+
+    def __init__(self, args_parser=NLIForZeroShotArgumentHandler(), *args, **kwargs):
+        super().__init__(*args, args_parser=args_parser, **kwargs)
+
+    def __call__(self, sequences, candidate_labels, hypothesis_template="This text is about {}.", multi_class=False):
+        outputs = super().__call__(sequences, candidate_labels, hypothesis_template)
+        num_sequences = 1 if isinstance(sequences, str) else len(sequences)
+        num_labels = 1 if isinstance(candidate_labels, str) else len(candidate_labels)
```

Comment:
if I pass a list of one candidate label, should `is_multiclass` be True?

Reply:
Yes, if they pass a single label, either as a list or a string, expected behavior is to get the unnormalized probability of that label, otherwise the output would always be 1.